simple/deployment.yaml

```diff
@@ -0,0 +1,3479 @@
+                 UM_RESOURCE_ID VARCHAR(255) NOT NULL,
+                 UM_ACTION VARCHAR(255) NOT NULL,
+                 UM_TENANT_ID INTEGER DEFAULT 0,
+    		UM_MODULE_ID INTEGER DEFAULT 0,
+    			       UNIQUE(UM_RESOURCE_ID,UM_ACTION, UM_TENANT_ID),
+                 PRIMARY KEY (UM_ID, UM_TENANT_ID)
+    )ENGINE INNODB;
+
+    CREATE INDEX INDEX_UM_PERMISSION_UM_RESOURCE_ID_UM_ACTION ON UM_PERMISSION (UM_RESOURCE_ID, UM_ACTION, UM_TENANT_ID);
+
+    CREATE TABLE UM_ROLE_PERMISSION (
+                 UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+                 UM_PERMISSION_ID INTEGER NOT NULL,
+                 UM_ROLE_NAME VARCHAR(255) NOT NULL,
+                 UM_IS_ALLOWED SMALLINT NOT NULL,
+                 UM_TENANT_ID INTEGER DEFAULT 0,
+    	     UM_DOMAIN_ID INTEGER,
+                 UNIQUE (UM_PERMISSION_ID, UM_ROLE_NAME, UM_TENANT_ID, UM_DOMAIN_ID),
+    	     FOREIGN KEY (UM_PERMISSION_ID, UM_TENANT_ID) REFERENCES UM_PERMISSION(UM_ID, UM_TENANT_ID) ON DELETE CASCADE,
+    	     FOREIGN KEY (UM_DOMAIN_ID, UM_TENANT_ID) REFERENCES UM_DOMAIN(UM_DOMAIN_ID, UM_TENANT_ID) ON DELETE CASCADE,
+                 PRIMARY KEY (UM_ID, UM_TENANT_ID)
+    )ENGINE INNODB;
+
+    -- REMOVED UNIQUE (UM_PERMISSION_ID, UM_ROLE_ID)
+    CREATE TABLE UM_USER_PERMISSION (
+                 UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+                 UM_PERMISSION_ID INTEGER NOT NULL,
+                 UM_USER_NAME VARCHAR(255) NOT NULL,
+                 UM_IS_ALLOWED SMALLINT NOT NULL,
+                 UM_TENANT_ID INTEGER DEFAULT 0,
+                 FOREIGN KEY (UM_PERMISSION_ID, UM_TENANT_ID) REFERENCES UM_PERMISSION(UM_ID, UM_TENANT_ID) ON DELETE CASCADE,
+                 PRIMARY KEY (UM_ID, UM_TENANT_ID)
+    )ENGINE INNODB;
+
+    -- REMOVED UNIQUE (UM_PERMISSION_ID, UM_USER_ID)
+    CREATE TABLE UM_USER_ROLE (
+                 UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+                 UM_ROLE_ID INTEGER NOT NULL,
+                 UM_USER_ID INTEGER NOT NULL,
+                 UM_TENANT_ID INTEGER DEFAULT 0,
+                 UNIQUE (UM_USER_ID, UM_ROLE_ID, UM_TENANT_ID),
+                 FOREIGN KEY (UM_ROLE_ID, UM_TENANT_ID) REFERENCES UM_ROLE(UM_ID, UM_TENANT_ID),
+                 FOREIGN KEY (UM_USER_ID, UM_TENANT_ID) REFERENCES UM_USER(UM_ID, UM_TENANT_ID),
+                 PRIMARY KEY (UM_ID, UM_TENANT_ID)
+    )ENGINE INNODB;
+
+    CREATE TABLE UM_SHARED_USER_ROLE(
+        UM_ROLE_ID INTEGER NOT NULL,
+        UM_USER_ID INTEGER NOT NULL,
+        UM_USER_TENANT_ID INTEGER NOT NULL,
+        UM_ROLE_TENANT_ID INTEGER NOT NULL,
+        UNIQUE(UM_USER_ID,UM_ROLE_ID,UM_USER_TENANT_ID, UM_ROLE_TENANT_ID),
+        FOREIGN KEY(UM_ROLE_ID,UM_ROLE_TENANT_ID) REFERENCES UM_ROLE(UM_ID,UM_TENANT_ID) ON DELETE CASCADE,
+        FOREIGN KEY(UM_USER_ID,UM_USER_TENANT_ID) REFERENCES UM_USER(UM_ID,UM_TENANT_ID) ON DELETE CASCADE
+    )ENGINE INNODB;
+
+    CREATE TABLE UM_ACCOUNT_MAPPING(
+    	UM_ID INTEGER NOT NULL AUTO_INCREMENT,
+    	UM_
```

Comment:
@NishikaDeSilva Can you please update the Kubernetes Deployment API version as in [1]?

[1]: [Kubernetes API Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deployment-v1-apps)